Makefile

```diff
@@ -73,7 +73,7 @@ style-fix:
 		echo "Refusing to run on dirty git state. Commit your changes first."; \
 		exit 1; \
 	fi; \
-	git ls-files | grep -e '\.\(c\|h\|cpp\)$$' | xargs clang-format-6.0 -i -style=file;
+	git ls-files | grep -e '\.\(c\|h\|cpp\)$$'| grep -v '^tests\/' | xargs clang-format-6.0 -i -style=file;
```

Comment:
After some online searching I found a better solution for clang-formatting the tests: https://github.com/cpputest/cpputest/issues/1080#issuecomment-600154657
Adding the `TypenameMacros: ['TEST']` and using clang-format-9 clang-format plays nice with the tests :slightly_smiling_face: